spec/features/messages_features_spec.rb

```diff
@@ -0,0 +1,22 @@
+require './app.rb'
+require 'spec_helper'
+require 'pg'
+
+feature 'a user can post a message to chitter' do
+  scenario 'a user post a message which is shown back to them' do
+    connection = PG.connect :dbname => 'chitter_test'
```

Comment:
At this stage, you've repeating this process a lot, you could always define the database connections bases on the environment variable. (though maybe that's something you would of done at a later stage)